packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -0,0 +1,48 @@
+import { Page } from 'playwright';
+
+const staticSelectors = {
+	modalContinueButton: 'button:text("Continue")',
+};
+
+const dynamicSelectors = {
+	cartItem: ( itemName: string ) =>
+		`[data-testid="review-order-step--visible"] .checkout-line-item >> text=${ itemName.trim() }`,
+	removeCartItemButton: ( itemName: string ) =>
+		`[data-testid="review-order-step--visible"] button[aria-label*="Remove ${ itemName.trim() } from cart"]`,
+};
```

Comment:
Interesting, I have not seen this before, but definitely useful in other situations I've faced.

Question: is it possible to have it all grouped under `const selectors = { ... }`? Or does it have to be a separate constant?